Diary/View/DiaryList/DiaryListViewController.swift

```diff
@@ -107,10 +146,10 @@ final class DiaryListViewController: UIViewController {
 
 extension DiaryListViewController: UITableViewDelegate {
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
-        guard let data = diaryViewModel.diaryContents?[indexPath.row] else {
+        guard let data = diaryViewModel?.diaryContents?[indexPath.row] else {
```

Comment:
```swift
public extension Collection {
  /// Returns the element at the specified index iff it is within bounds, otherwise nil.
  subscript (safe index: Index) -> Element? {
    return indices.contains(index) ? self[index] : nil
  }
}

let ... = someCollection[safe: index]
```

이렇게 좀 더 안전하게 인덱스에 접근할 수 있어요!